spec/models/issues/people/duplicated_document_spec.rb

```diff
@@ -51,10 +52,13 @@
       it "is not fixed for the existing person" do
         expect { subject }.not_to change { issue.fixed_for?(existing_person) }.from(false)
       end
+
+      it_behaves_like "an event notifiable with hutch"
```

Comment:
Last spec comment, very minor and just a personal style not necessarilly the best, feel free to just ignore it and move on...

I usually "pass" the variable needed by shared specs like this:

```ruby
it_behaves_like "an event notifiable with hutch" do
  let(:publish_notification) { ["census.people.full_status_changed", { person: existing_person.qualified_id }] }
end
```

I like it because it makes the context needed by the shared spec very clear.

Reply:
great, I was looking for that and I didn't know how to do it :+1: